### ISSUE TITLE
Right clicking the desktop crashes sway

### ISSUE THREAD

**Issue by SirCmpwn** [SirCmpwn/sway#68]
cc @Luminarys

**Comment by Luminarys** [comment]
I can't seem to duplicate this. Where does gdb say it's crashing, and why?

**Comment by SirCmpwn** [comment]
You have the latest code? Not able to run it in gdb right now.

**Comment by Luminarys** [comment]
I do, but right clicks on the desktop don't seem to do anything.

**Comment by Ninja3047** [comment]
I've been able to reproduce this by spamming right click while moving the mouse.

**Comment by Luminarys** [comment]
I'll get around to testing this quickly, in the meanwhile, would you be able to provide some info from gdb or logs?

**Comment by Ninja3047** [comment]
```
focus:R|(0x60c6c0)(p:(nil))(f:0x14d61a0)(h:-1)Type:Root|layout:NONE|w:0|h:0|x:0|y:0|vis:f|wgt:0|name:(null)|children:1
(
 focus:X|(0x14d61a0)(p:0x60c6c0)(f:0x1503800)(h:1)Type:Output|layout:NONE|w:681|h:746|x:0|y:0|vis:f|wgt:1|name:None-0|children:1
 (
  focus:F|(0x1503800)(p:0x14d61a0)(f:(nil))(h:-1)Type:Workspace|layout:Horiz|w:681|h:746|x:0|y:0|vis:t|wgt:1|name:1|children:0
   )
)
wlc: Xwayland started (DISPLAY :1)
wlc: xfixes (5.0)
Compositor is ready, executing cmds in queue
Mouse button 273 pressed
Setting focus to 0x1503800:-1
Mouse button 273 released
Mouse button 273 pressed
Setting focus to 0x1503800:-1
wlc: Parent exit (7322)
```
The logging says when it exits.

**Comment by Luminarys** [comment]
Ok, I've been able to replicate the bug, thanks.

**Comment by SirCmpwn** [comment]
aside: #66 needs to happen

**Comment by Luminarys** [comment]
Fixed the issue, will be pushing in upcoming commit.

**Comment by Luminarys** [comment]
Alright, fixed with 23b90d8e69d203697c5548fdd140bf3052749dfa.
